apps/meteor/server/models/raw/LivechatDepartmentAgents.ts

```diff
@@ -152,7 +152,7 @@ export class LivechatDepartmentAgentsRaw extends BaseRaw<ILivechatDepartmentAgen
 	}
 
 	removeByDepartmentId(departmentId: string): Promise<DeleteResult> {
-		return this.deleteOne({ departmentId });
+		return this.deleteMany({ departmentId });
```

Comment:
this is a bug fix btw.. may need to push as a separate PR

Reply:
What was the bug? 👀

Reply:
if you delete a department with multiple agents assigned, only one agent was actually removed from the collection, so a bit of garbage would be stored in the database..

Reply:
ouch